src/main.cpp

```diff
@@ -66,7 +72,7 @@ int main ( int argc, char** argv )
 
     // initialize all flags and string which might be changed by command line
     // arguments
-#if defined( SERVER_BUNDLE ) && ( defined( Q_OS_MACX ) )
+#if defined( SERVER_BUNDLE ) && defined( Q_OS_MACX )
     // if we are on MacOS and we are building a server bundle, starts Jamulus in server mode
     bool bIsClient = false;
```

Comment:
What happens for GUI serveronly builds on Windows? Why don’t we need to set this variable

Reply:
`SERVER_BUNDLE` isn't `SERVER_ONLY` - it's creating a _full_ Jamulus instance.  However, rather than start it with `--server`, it's forced here because of the build option.

We don't do that on Windows - we provide two shortcuts, one with and one without `--server`.